memo/README.rst

```diff
@@ -0,0 +1,28 @@
+ics_doc/memo
+======
+
+This directory stores bluk (non-organized) memos and/or FAQs which could be 
+useful for running or development of tron and actors 
+(refer `INSTRM-284 <https://pfspipe.ipmu.jp/jira/browse/INSTRM-284>`_). 
+
+Add new memo into files in this directory listed below, or add a new file 
+with adding a short description into following list. 
```

Comment:
shall state that this directory is for unorganized memos, and does not require any review process to master.